utilsProcessing.py

```diff
@@ -35,8 +38,21 @@ def lowPassFilter(time, data, lowpass_cutoff_frequency, order=4):
 
     return dataFilt
 
+# %% Segment gait
+def segment_gait(session_id, trial_name, data_folder, gait_cycles_from_end=0):
```

Comment:
@suhlrich I don't think it is great to have a different structure for this function compared to the other ones. In the other ones, we pass the pass to the already downloaded trial, whereas here we download the file within the function, which is confusing. I understand it is nice to have everything contained. But this function should segment a gait trial, not download and segment.

Reply:
good point. I removed the download from the function but left the segmentation code in the class. I think that's generally where it belongs. If we create a squats and STS class, I think we should move the actual segmentation code to the class as well. I made comments of this in the code.